build.gradle

```diff
@@ -409,7 +409,7 @@ project('loan-broker') {
 project('loanshark') {
 	description = 'Loan Shark Sample'
 
-	apply plugin: 'jetty'
+	apply plugin: 'org.akhikhl.gretty'
```

Comment:
Actually, it doesn't look like the original jetty config works; reverted; we can revisit this later.